receiver/mysqlreceiver/README.md

```diff
@@ -14,12 +14,11 @@ Collecting most metrics requires the ability to execute `SHOW GLOBAL STATUS`. Th
 
 ## Configuration
 
-The following settings are required to create a database connection:
-- `endpoint`
-- `username`
-- `password`
 
 The following settings are optional:
+- `endpoint`: (default = `localhost:3306`)
+- `username`: (default = `root`)
+- `password`: The password to the username. If not specified, root/admin access is required.
```

Comment:
> If not specified, root/admin access is required.

Is this necessarily true? Obviously not recommended, but could you have a non-root user w/o a password?

Reply:
actually yes, found out if you set authentication to `auth_socket` you can just use the unix socket endpoint. I'll update it.